boards/renesas/ek_ra4m3/Kconfig.board

```
@@ -0,0 +1,6 @@
+# Copyright (c) 2024 Piotr Rak <piotr.rak@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_RA4M3_EK
+	bool "Renesas EK-RA4M3 board"
+	depends on SOC_R7FA4M3AF3CFB
```

And need to select the soc e.g.
```suggestion
	select SOC_R7FA4M3AF3CFB
```